alphalens/performance.py

```diff
@@ -39,10 +39,10 @@ def factor_information_coefficient(factor,
     forward_returns : pd.DataFrame - MultiIndex
         Forward returns in indexed by date and asset.
         Separate column for each forward return window.
-    sector_adjust : bool
-        Demean forward returns by sector before computing IC.
-    by_sector : bool
-        If True, compute daily IC separately for each sector.
+    group_adjust : bool
```

Comment:
At this point we assume you've already added the group to the MultiIndex. This isn't mentioned in the docstring and should be.